package.json

```diff
@@ -26,7 +26,7 @@
     "json-editor": "0.7.28",
     "json-loader": "0.5.4",
     "moment": "2.14.1",
-    "node-sass": "3.9.1",
+    "node-sass": "3.9.3",
```

Comment:
This was on master, and was needed to get the tests to pass. I can probably get rid of it with some rebasing.